app/models/http_proxy.rb

```diff
@@ -16,8 +16,7 @@ class HttpProxy < ApplicationRecord
 
   validates :name, :presence => true, :uniqueness => true
 
-  validates :url, :presence => true
-  validates :url, :format => URI.regexp(["http", "https"])
+  validates :url, :format => { :with => /\Ahttps?:\/\// }, :presence => true
```

Comment:
now, we are not validating the rest of the url right? `http://666ž` is valid now, right? :thinking:

Reply:
Nope, we are not validating the rest and `http://666ž` passes the validation before the change as well.

Reply:
I don't think it's true... the URI regexp is quite sofisticated. see https://www.rubydoc.info/stdlib/uri/1.8.6/URI/REGEXP/PATTERN the `X_ABS_URI` is the one that is used if I read correctly :)

Reply:
But nvm :+1: